core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/item/Aggregators.java

```diff
@@ -1048,6 +1048,31 @@ public SetConsumer<T> newItemConsumer(IType<IItem> itemType) {
 		return filter(distinctAsString(attribute, ", "), ItemFilters.type(typeId)); //$NON-NLS-1$
 	}
 
+	public static <T> IAggregator<Number, ?> getIdentifier(String typeId, IAttribute<T> attribute) {
```

Comment:
Do we need a special aggregator for this? We should override the value type for the PID field so that the correct value type is returned.